mlflow/projects/_project_spec.py

```diff
@@ -99,7 +99,10 @@ def get_entry_point(self, entry_point):
         if entry_point in self._entry_points:
             return self._entry_points[entry_point]
         _, file_extension = os.path.splitext(entry_point)
-        ext_to_cmd = {".py": "python", ".sh": os.environ.get("SHELL", "bash")}
+        ext_to_cmd = {
+            ".py": os.environ.get("PYTHON", "python3"),
```

Comment:
Is this a best practice for setting an environment variable for which python to use? I just want to make sure this doesn't stomp on or miss some existing convention.